katello/katello-restore

```diff
@@ -31,39 +42,115 @@ end
 def reset_katello
   puts "Resetting Katello"
   `tar --selinux --overwrite --listed-incremental=/dev/null -xzf config_files.tar.gz -C /`
-  `foreman-installer --scenario katello --reset`
+  installer = "yes | foreman-installer -v --reset"
+  if @foreman_proxy_content
+    installer << " --scenario #{FOREMAN_PROXY_CONTENT} --foreman-proxy-register-in-foreman false"
+  else
+    installer << " --scenario katello"
+  end
+  installer << " --disable-system-checks" if @disable_system_checks
+  puts installer
+  installer_output = `#{installer}`
+  puts installer_output
+
   puts "Done.\n"
 end
 
+def restore_psql_dumps
+  puts "Restoring postgres dump files"
+  `katello-service start --only postgresql`
+  `runuser - postgres -c "dropdb foreman"`
+  `runuser - postgres -c "dropdb candlepin"`
+  `runuser - postgres -c "psql -f #{DIR}/pg_globals.dump postgres 2>/dev/null"`
+  `runuser - postgres -c "pg_restore -C -d postgres #{DIR}/foreman.dump"`
+  `runuser - postgres -c "pg_restore -C -d postgres #{DIR}/candlepin.dump"`
+  `katello-service stop --only postgresql`
+  puts "Done."
+end
+
+def migrate_pulp
+  puts "Migrating pulp databases"
+  necessary_services = "mongod,qpidd"
+  pulp_services = "pulp,celerybeat,pulp_workers,pulp_resource_manager"
+  `katello-service start --only #{necessary_services}`
+  `katello-service stop --only #{pulp_services}`
+  `sudo -u apache pulp-manage-db`
+  `katello-service start`
```

Comment:
Let's remove this here. We restart services after the restore_databases method is called, no need to start services 2x.